### ISSUE TITLE
Add internal launch API endpoint

### ISSUE THREAD

**Issue by jmandel** [smart-on-fhir/smart-launcher-v2#23]
We discussed adding a CDS hooks radio button back to the v2 launcher and decided that could wait...

But on review, there is one smaller feature that would be helpful to add: an internal API to support https://github.com/cds-hooks/sandbox/blob/8666de9fa6bea7b3a7bee5e79c76768703cbbf51/src/retrieve-data-helpers/launch-context-retrieval.js#L37 in cases where the existing CDS hooks sandbox needs to trigger a launch independently.

Basically this is an endpoint that takes an arbitrary set of parameters (patient, app context, etc) and returns a launch id that binds those parameters, so if an EHR launch is eventually completed using this launch id, the access token response will include those parameters.

**Comment by janstadt** [comment]
@jmandel is this repo still active? Just curious if there are other similar libs that i could look at for emulating an EHR as i have a need to test out cds hooks and a smart launch app. I've had decent luck with https://sandbox.logicahealth.org/ but it seems as though its no longer being supported as well as https://sandbox.cds-hooks.org/ but i'd like to see if there is something a bit more like this repo that i can leverage. TIA!

**Comment by vlad-ignatov** [comment]
I've started working on CDS Hook launch support, but I find it confusing so I want to lay out the flow I am considering and verify that we are on the same page before I commit anything. Here is my current plan:
1. Have a "CDS Hooks Launch" option in the Launch Type drop-down. If selected, offer a basic UI for selecting a patient, practitioner, etc.
2. The Launch URL field at the bottom can point to the launch url of a running CDS sandbox. Defaults to `https://sandbox.cds-hooks.org/launch.html` but people can change it if they use another instance (including one on localhost)
3. Clicking on the Launch button launches the CDS sandbox in new tab with our FHIR server and selected patient
4. From there it is up to the user to create new services and register them with the selected CDS sandbox

Is that good enough, or did you expect something else? @jmandel?
